cloud_formation/configs/api.py

```diff
@@ -441,8 +441,25 @@ def update(session, domain):
     return success
 
 def delete(session, domain):
-    names = AWSNames(domain)
-    aws.route53_delete_records(session, domain, names.endpoint)
-    aws.sqs_delete_all(session, domain)
-    aws.policy_delete_all(session, domain, '/ingest/')
-    CloudFormationConfiguration('api', domain).delete(session)
+    yes = {'yes','y', 'ye', ''}
```

Comment:
could also just check the first character of the response

Reply:
Will do